packages/vitest/src/runtime/runners/test.ts

```diff
@@ -62,6 +62,14 @@ export class VitestTestRunner implements VitestRunner {
       }
 
       const result = await this.snapshotClient.finish(suite.file.filepath)
+      if (
+        this.workerState.config.snapshotOptions.updateSnapshot === 'none'
+        && result.unchecked
+      ) {
+        const error = new Error(`Obsolete snapshots found: ${result.uncheckedKeys.join(', ')}`)
```

Comment:
I wonder if we can print it in a better way (should we? obsolete snapshots are already printed one line at a time, but without the separation by test file)

Reply:
I tweaked the error message. Please check the example in the PR description. It's mostly a copy of the base reporter.